safe-global/safe-wallet-web
# fix: Use PNG instead of SVG for Redefine logo

## What it solves

Resolves #2314 

## How this PR fixes it

- Uses PNG instead of SVG

## How to test it

1. Create a transaction on mainnet
2. Observe the Redefine logo looking the same as before in both dark and light mode

## Screenshots
<img width="696" alt="Screenshot 2023-08-08 at 10 17 32" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/dbadb3e0-1682-4e22-a6d5-66214ec92afa">
<img width="711" alt="Screenshot 2023-08-08 at 10 17 39" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/40ffcbb8-176e-4d6f-8edd-7de400489d75">

## Checklist
